client/blocks/nav-unification-quick-switch-modal/style.scss

```
@@ -0,0 +1,241 @@
+// @TODO: remove the ignore rule and replace font sizes accordingly
+/* stylelint-disable scales/font-size */
+
+@import '@automattic/typography/styles/fonts';
+@import '@automattic/onboarding/styles/mixins';
+@import '@wordpress/base-styles/breakpoints';
+
+$modal-breakpoint: 600px;
+$modal-breakpoint-phone-height: 415px;
+$modal-padding-v: 40px;
+$modal-padding-h: 50px;
+$modal-content-min-height: 410px;
+$modal-content-min-height-landscape: 370px;
+$modal-footer-padding-v: 20px;
+$modal-footer-height: 30px + ( $modal-footer-padding-v * 2 );
+
+// Core modal style overrides
+.nav-unification-quick-switch-modal {
+	&.components-modal__frame {
+		overflow: visible;
+		height: 65vh;
+		top: calc( 17.5vh - #{$modal-footer-height / 2} );
+
+		@media ( max-width: $modal-breakpoint ) {
+			width: 90vw;
+			min-width: 90vw;
+			left: 5vw;
+			right: 5vw;
+		}
+
+		@media ( min-width: $modal-breakpoint ) {
+			width: 740px;
+			height: $modal-content-min-height;
+			top: calc( 50% - #{$modal-footer-height / 2} );
+		}
+
+		@media ( min-width: $modal-breakpoint ) and ( max-height: $modal-breakpoint-phone-height ) {
+			height: $modal-content-min-height-landscape;
+			top: calc( 50% - 0px );
```

Why is `calc` needed instead of the following?
```suggestion
			top: 50%;
```

---

Will remove, this was left over from the nav unification modal and amended during development.